decidim-debates/spec/commands/decidim/debates/admin/update_debate_spec.rb

```diff
@@ -62,4 +68,23 @@
       expect(action_log.version.event).to eq "update"
     end
   end
+
+  context "when everything is ok with attachments" do
```

Comment:
Could you add a spec with what happens what if you upload a document that is not allowed?